packages/design/src/TopNav/TopNavUserMenu/TopNavUserMenu.jsx

```diff
@@ -93,7 +99,9 @@ const StyledAvatar = styled.div`
   justify-content: center;
   height: 32px;
   margin-left: 16px;
-  width: 32px;
+  width: 100%;
+  max-width: 32px;
+  min-width: 32px;
```

Comment:
Fixes warping of perfect circle when user name is too long

from:
![Screenshot from 2020-06-29 22-02-26](https://user-images.githubusercontent.com/43280172/86085555-46703780-ba54-11ea-9a1a-5471521610f4.png)

to:
![Screenshot from 2020-06-29 21-58-29](https://user-images.githubusercontent.com/43280172/86085495-26d90f00-ba54-11ea-824a-0476b814fc9c.png)